drivers/include/lsm6ds3.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
```

Comment:
Add HAW Hamburg here and let FU Berlin in, as you code seems to base on Thomas Eichingers code